docs/faq/lukso/wallet-support.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_label: 'Wallet Support'
+sidebar_position: 5
+---
+
+# Wallet Support
+
+### Whats the best way to use LUKSO?
+
+It's suggested to use the [Universal Profile Extension](https://docs.lukso.tech/guides/browser-extension/install-browser-extension) while interacting with apps and services on LUKSO. The extension offers you a much more user-friendly and feature-rich experience than you would get using regular wallets.
```

Comment:
For links within the docs, please use relative link, not absolute links. This helps detecting broken links by docusaurus.